views/partials/navbar.hbs

```diff
@@ -50,15 +50,15 @@
 
             {{!-- Uncomment ahead of hackathon start --}}
             {{!-- <!-- PARTS -->
-            <li><a href="/dashboard/parts">Parts</a></li>
+            <li><a href="/dashboard/parts">Parts</a></li> --}}
 
-            <!-- MY PARTS -->
-            <li><a href="/dashboard/me/parts">My Parts</a></li>
+            {{!-- <!-- MY PARTS -->
+            <li><a href="/dashboard/me/parts">My Parts</a></li> --}}
 
             <!-- MY TEAM -->
-            <li><a href="/dashboard/me/parts">My Team</a></li>
+            <li><a href="/dashboard/me/team">My Team</a></li>
 
-            <!-- RESOURCES -->
+            {{!-- <!-- RESOURCES -->
```

Comment:
Can you undo all your changes to this file, both here and lower down in the file? Right now it conflicts with the development branch, and we don't want this page exposed to users until the hackathon starts